plugins/rapid7_intsights/plugin.spec.yaml

```diff
@@ -615,3 +615,114 @@ actions:
         description: New created alert ID
         type: string
         required: true
+triggers:
+  new_alert:
+    title: New Alert
+    description: Run when a new alert that matches the given criteria is created in Intsights
+    input:
+      alert_type:
+        title: Alert Type
+        description: 'List of alert types. Allowed values: AttackIndication, DataLeakage, Phishing, BrandSecurity, ExploitableData, vip'
+        type: '[]string'
+        required: false
+        example: ["Phishing"]
+      severity:
+        title: Severity
+        description: 'List of alerts severity. Allowed values: High, Medium, Low'
+        type: '[]string'
+        required: false
+        example: ["Low"]
+      source_type:
+        title: Source Type
+        description: 'List of alerts source type. Allowed values: Application Store, Cyber Security Blog, Hacking News, Cyber Crime Forum, Hacktivism Forum, Social Media, Facebook, Twitter, LinkedIn, Google Plus, VK, Vimeo, YouTube, IRC Channel, IOC Block List, Credit Card Black Market, Paste Site, Data Leakage Website, Leaked Database, File Sharing Website, Gray Hat Website, Black Market, WHOIS servers, Company Website, Wikileaks, Pinterest, Tumblr, Instagram, Telegram, Webmail, Malware Analysis, Firehol, VRA, Other'
+        type: '[]string'
+        required: false
+        example: ["Application Store"]
+      network_type:
+        title: Network Type
+        description: 'List of network type. Allowed values: ClearWeb, DarkWeb'
+        type: '[]string'
+        required: false
+        example: ["DarkWeb"]
+      matched_asset_value:
+        title: Matched Asset Value
+        description: List of matched asset values
+        type: '[]string'
+        required: false
+        example: ["example.com"]
+      remediation_status:
+        title: Remediation Status
+        description: 'List of remediation statuses. Allowed values: InProgress, Pending, CancellationInProgress, Cancelled, CompletedSuccessfully, Failed'
+        type: '[]string'
+        required: false
+        example: ["InProgress", "Pending"]
+      source_date_from:
+        title: Source Date From
+        description: Start date to fetch from in Unix Millisecond Timestamp
+        type: string
+        required: false
+        example: 1633047083142
+      source_date_to:
+        title: Source Date To
+        description: End date to fetch to in Unix Millisecond Timestamp
+        type: string
+        required: false
+        example: 1633047102456
+      found_date_from:
+        title: Found Date From
+        description: Start date to fetch from in Unix Millisecond Timestamp
+        type: string
+        required: false
+        example: 0
+      found_date_to:
+        title: Found Date To
+        description: End date to fetch to in Unix Millisecond Timestamp
+        type: string
+        required: false
+        example: 1633047102456
```

Comment:
Getting a different description source vs found dates here would be useful. What does "source" mean and what is "found"?

Reply:
I changed here description, but in documentation there is no more information about this field. Could we contact with intsight team and ask which value can be here?